modules/util/portal-tools-db-schema-importer/src/main/java/com/liferay/portal/tools/db/schema/importer/DBImporter.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.tools.db.schema.importer;
+
+import com.liferay.portal.tools.db.schema.importer.jdbc.util.DataSourceFactoryUtil;
+
+import javax.sql.DataSource;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Options;
+
+/**
+ * @author Mariano Álvaro Sáiz
+ */
+public class DBImporter {
```

Comment:
The main class should be called DBSchemaImporter, we need to figure out other name for the other class

Reply:
Maybe we can call the other class DBSchemaImporterUtil and make it static passing the parameters to the importDB method.

Anyways, let's send the first PR with the skeleton and only the class DBSchemaImporter with the main and the parameters. After that, we can decide the rest.

Reply:
What about calling main class DBSchemaImporter and the other one DBSchemaImporterHelper so that we don't need to make it static?

Reply:
DBSchemaImporterHelper is also a good too :+1: